projects/kit/components/toast/toast.component.ts

```diff
@@ -0,0 +1,14 @@
+import {ChangeDetectionStrategy, Component, ViewEncapsulation} from '@angular/core';
+
+@Component({
+    standalone: true,
+    selector: 'tui-notification-toast',
```

Comment:
Looking at the specs it feels like we should be able to put it on interactive elements like buttons or links. Let's change it to attribute selector.

Reply:
fixed